### PR TITLE
PMM-11445 edit page missing

### PR DESCRIPTION
**Pull request by rasika-chivate**
Edit page missing on PMM docs

### CODE REVIEW

**Comment by LenzGr** [comment]
@rasika-chivate please consider adding some background info to the git commit message about this change, and why it is necessary. I added the following to mine:
```

Material Theme for MkDocs version 9.x changed the default behavior of the
"Edit this page" functionality. Now both editing and viewing the source code
must be explicitly enabled:

https://squidfunk.github.io/mkdocs-material/upgrade/#upgrading-from-8x-to-9x
```

**Comment by atymchuk** [comment]
@rasika-chivate @LenzGr Why don't we pin the MaterialUI version so we 1) upgrade when necessary and 2) don't face surprises like this.

**Review by atymchuk — APPROVED** [review]

File: mkdocs.yml [hunk 1]
```diff
@@ -6,6 +6,15 @@ site_url: https://docs.percona.com/percona-monitoring-and-management/
 plugins:
   section-index: {}
 
+features:
```
Comment (atymchuk) [inline mkdocs.yml:9]: I believe this changes belongs in mkdocs-base.yml, since it is extended by multiple yml files.

**Review by atymchuk — CHANGES_REQUESTED** [review]

**Review by atymchuk — APPROVED** [review]

**Comment by LenzGr** [comment]
@rasika-chivate seems like the edit icons re-appeared on the PMM docs, even though this PR hasn't been merged. Could you please take a look?

**Comment by rasika-chivate** [comment]
Modified the docs with the PR: https://github.com/percona/pmm-doc/pull/972

**Pull request closed without merging**